tools/c7n_azure/c7n_azure/storage_utils.py

```diff
@@ -76,6 +76,14 @@ def create_queue_from_storage_account(storage_account, name, session):
             token_credential=token)
         return queue_service.create_queue(name)
 
+    @staticmethod
+    def delete_queue_from_storage_account(storage_account, name, session):
```

Comment:
nit: feels weird.. Session static method that requires you to pass session object.

Reply:
In this case I just copied the identical signature from the `create` above so the `delete` matches.  That said I wrote the `create` also a long time ago, if I recall we take them as parameters because in Mailer we don't have sessions in quite the same way.     Not Session static though, storage static.